content/en/observability_pipelines/quickstart/_index.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Quickstart
```

Comment:
Is the difference between the Quickstart and Installation doc that the Quickstart is for Datadog users?

Reply:
Yes. The Installation doc is very generic and doesn't really leave the user with a starting configuration that will do what they want.